packages/as-transformer/src/transformers/testTable.ts

```diff
@@ -8,181 +8,96 @@ import { SimpleParser } from 'visitor-as';
-    // The function signature is the following:
-    // testTable(<test set name>, <template of value to compute>,
-    // <comparison criterion>, <template of expected result>, <test failure strategy>, [<test values>...])
-    // <template of expected result>> field is optional
-    const args: string[] = [];
-    node.args.forEach((element) => {
-      const content = element.range.source.text.slice(
-        element.range.start,
-        element.range.end,
-      );
-      args.push(content);
-    });
-
-    const testSetName = args[0];
-    const gotTemplate = args[1]!;
-    const comparisonCriterion = args[2]!;
-    let expectedTemplate = args.length == 5 ? args[3]! : '';
+    // Extract node arguments as text
+    const args = node.args.map((arg) =>
+      arg.range.source.text.slice(arg.range.start, arg.range.end),
+    );
 
-    // extracts all test values and converts potential breaking single quotes to double ones.
+    const testName = args[0];
+    const expectationTemplate = args[1];
     const rawValues = (node.args[args.length - 1] as ArrayLiteralExpression)
       .elementExpressions;
     const values = rawValues.map((e) =>
       e.range.source.text.slice(e.range.start, e.range.end).replace(/'/g, '"'),
     );
 
-    let expr = `describe(${testSetName}, () => {\n`;
-
-    const { ifExpr, hasWant: needWant } =
-      TestTable.generateIfExpression(comparisonCriterion);
+    let expr = `describe(${testName}, () => {\n  `;
 
-    if (needWant && expectedTemplate == '') {
-      expectedTemplate = 'arg0';
-    }
-
-    let gotExpr;
-    let expectExpr;
-
-    // two concepts here:
-    // - testCounter keeps track of the number of tests
-    // - iValue is the index of the values array
-    // One test can involve one or multiple values.
-    // Here we loop until all test values are used.
+    let expectation;
     let testCounter = 0;
     for (let iValue = 0; iValue < values.length; testCounter++) {
-      // Destructuring assignment unpacks returned object values into distinct values.
-      ({ instantiation: gotExpr, iValue } = TestTable.hydrateTemplate(
-        gotTemplate,
-        values,
-        iValue,
-      ));
-      ({ instantiation: expectExpr, iValue } = TestTable.hydrateTemplate(
-        expectedTemplate,
+      ({ instantiation: expectation, iValue } = TestTable.hydrateTemplate(
+        expectationTemplate!,
         values,
         iValue,
       ));
-
-      const test = TestTable.generateTest(
-        testCounter.toString(), // use test counter as test name
-        gotExpr.replace(/['`"]/g, ''),
-        expectExpr.replace(/['`"]/g, ''),
-        ifExpr,
-        needWant,
-      );
-
-      expr += test.replace(/\n/g, '\n  '); // adds static indentation
+      const test = TestTable.generateTest(testCounter.toString(), expectation);
+      expr += test.replace(/\n/g, '\n  ');
     }
+
     expr += `\n});`;
 
+    // console.log(expr);
```

Comment:
to delete!

Reply:
Tbh it super usefull to debug as there is no way to get the result of transformed source...